src/components/Search/Search.scss

```diff
@@ -124,6 +132,17 @@
       display: none;
     }
 
+    .wkp-search-toggle-container--open {
+      .wkp-search-input {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        border: 2px solid #b7b7b7;
+        background-color: white;
+        width: 100%;
+      }
+    }
```

Comment:
What is different with lines from line 10 ?

Reply:
Yes, this can be simplified. will do